openapi/atlas-api-transformed.yaml

```diff
@@ -31146,7 +31334,8 @@ components:
           type: string
           description: MongoDB Cloud computes the current metric value as an average.
         operator:
-          $ref: "#/components/schemas/Operator"
+          type: string
```

Comment:
Operator was a string so it's fine to use it a string directly, no breaking change